kanidmd/src/lib/core/https.rs

```diff
@@ -1155,30 +1149,32 @@ pub fn create_https_server(
     opt_tls_params: Option<&TlsConfiguration>,
     role: ServerRole,
     cookie_key: &[u8; 32],
+    bundy_key: &str,
     status_ref: &'static StatusActor,
     qe_w_ref: &'static QueryServerWriteV1,
     qe_r_ref: &'static QueryServerReadV1,
 ) -> Result<(), ()> {
     info!("WEB_UI_PKG_PATH -> {}", env!("KANIDM_WEB_UI_PKG_PATH"));
 
-    // Create the in memory fernet key
-    let fernet_handle = fernet::Fernet::new(&fernet::Fernet::generate_key()).ok_or_else(|| {
-        error!("Failed to generate fernet key");
+    let bundy_handle = bundy::hs512::HS512::from_str(bundy_key).map_err(|e| {
+        error!("Failed to generate bundy handle - {:?}", e);
```

Comment:
This is going to come up as  "what does this even mean?!?!?!" ticket in about .. a year. :)

Reply:
If this fails to generate there is going to be some "chaos" 😅